tests/robot-tests/tests/general_public/glossary_page.robot

```diff
@@ -29,16 +29,18 @@ Search for Pupil referral unit
     [Tags]    HappyPath
     user verifies accordion is closed    P
 
-    user enters text into element    id:pageSearchForm-input    Pupil referral unit
+    user enters text into element    id:pageSearchForm-input    Permanent exclusion
```

Comment:
Interested to know why it was failing with Pupil referral unit, as I don't believe the glossary page has changed, and I don't think we've had an issue with this before.

After clicking the search option I would expect to jump to that entry such that it is top of the page, but right now it seems like it jumps to the middle of the screen. Maybe that has changed?

Reply:
the pupil referral unit test data isn't present anymore. Assumed this had been removed as part of the new methodology work

Reply:
looks like a wrong id was being selected. This has been fixed now